master/usage/policy/secure-metrics.md

```diff
@@ -0,0 +1,428 @@
+---
+title: Securing Calico's Prometheus endpoints
+---
+
+## About securing access to {{site.prodname}}'s metrics endpoints
+
+When using {{ site.prodname }} with Prometheus metrics enabled, we recommend using network policy
+to limit access to {{ site.prodname }}'s metrics endpoints.
+
+This guide provides two example workflows for creating network policies to limit access
+to {{site.prodname}}'s Prometheus metrics. Choosing an approach depends on your requirements.
+
+- [Using a blacklist approach](#using-a-blacklist-approach)
+
+  This approach allows all traffic to your hosts by default, but lets you limit access to specific ports using
+  {{site.prodname}} policy. This approach allows you to restrict access to access to specific ports, while leaving other
+  host traffic unaffected.
+
+- [Using a whitelist approach](#using-a-whitelist-approach)
+
+  This approach denies traffic to and from your hosts by default, and requires that all
+  desired communication be explicitly allowed by a network policy. This approach is more secure because
+  only explicitly-allowed traffic will get through, but it requires you to know all the ports that should be open on the host.
+
+## Prerequisites
```

Comment:
Some people may just click the above links and never read the prereqs. I would suggest for this reason perhaps to move this section to line 9. Then add a heading before the bullets that is something like ## Choosing an approach